### DISCUSSION TITLE
Reaching Out for Help

### DISCUSSION THREAD

**Discussion by bhh32** [cosmic-utils/discussions#3] [General]
Is there a way we are able to reach out the other developers of this project for help? I have a question on something, but I don't particularly want it to be seen by everyone who can look at the repos.

**Comment by wiiznokes** [comment]
A question on what matter ?

**Comment by bhh32** [comment]
A development question. I have two actually.

**Comment by wiiznokes** [comment]
Maybe you should try the pop os matter most, there is a development channel, you will probably have more visibility
